tests/test_stac.py

```diff
@@ -91,3 +91,35 @@ def test_create_item_xml_grid_code(self):
         self.assertEqual(
             item.datetime, str_to_datetime("20200902") + timedelta(hours=16)
         )
+
+    # test stac on year 2020
+    # handles resource description with and without ".tif" extension
+    def test_create_item_xml_ext(self):
+        item = create_item(
+            "ca",
+            "2020",
+            test_data.get_path("data-files/m_3211605_ne_11_060_20200415.tif"),
+            test_data.get_path("data-files/m_3211605_ne_11_060_20200415_20200730.xml"),
+        )
+
+        self.assertEqual(item.id, "ca_m_3211605_ne_11_060_20200415")
+        self.assertEqual(
+            item.datetime, str_to_datetime("20200415") + timedelta(hours=16)
```

Comment:
Why isn't this comparing an exact datetime value instead of constructing it from a string and timedelta?

Reply:
item.datetime was originally constructed as dt = dt + timedelta(hours=16)

- An annotation states "UTC is +4 ET". I believe the 16 timedelta deals with the conversion between 1) military/standard (+12) and 2) UTC/ET (+4). 

- The purpose of the time delta is a bit unclear, since it doesn't seem like NAIP reports actual times (based on spot checks on a few items). Time stamps are generated from the date and therefore always 00:00:00. The timedelta conversion doesn't really seem to provide any meaningful. 

Let me know your thoughts.

Reply:
I think you can just to this, which is more clear: str_to_datetime("2020-04-15T16:00:00Z")

Reply:
updated  to -> datetime.strptime("2020-09-02T16:00:00Z", "%Y-%m-%dT%H:%M:%SZ")

assert equal failing with -> str_to_datetime("2020-04-15T16:00:00Z")